instfiles/openrc/xrdp-sesman

```diff
@@ -0,0 +1,4 @@
+#!/sbin/openrc-run
+
+command="/usr/local/sbin/xrdp-sesman"
```

Comment:
This shouldn't be hardcoded to '/usr/local' prefix.

Reply:
Ahh I see.  Yes, I believe there are some prefix replacements available in the other scripts, I'll look at this.